lsr_role2collection/collection_readme.txt

```diff
@@ -0,0 +1,36 @@
+﻿# Linux System Roles Ansible Collection
+Ansible collection for interacting with the Linux System Roles.

```

Comment:
I suggest adding information that it is Technology Preview: "Ansible collection for interacting with the Linux System Roles, available as a Technology Preview."

Reply:
That's for downstream/Automation Hub only.  That shouldn't go into this, the upstream doc.

Reply:
That's a bit weird description... the collection does not interact with our system roles, it contains them.